EventVisualisation/View/include/EventVisualisationView/EventManager.h

```diff
@@ -74,25 +73,23 @@ class EventManager final : public TEveEventManager, public TQObject
   void NextEvent() override;
   void PrevEvent() override;
   void Close() override;
+  void DisplayCurrentEvent();
```

Comment:
since you are anyway modifying old code, better at once to bring it to O2 coding conventions, with methods names starting by lower-case.

Reply:
ok